promscale/installation/helm.md

```diff
@@ -13,10 +14,12 @@ Before you install the TimescaleDB Helm chart, you need to configure these
 settings in the `values.yaml` configuration file:
 *   Credentials for the superuser, admin, and other users
 *   TLS Certificates
-*   **Optional:** `pgbackrest` configuration
+*   **Optional:** `pgbackrest` [configuration][timescale-backups]
 
 <highlight type="note">
-If you do not configure the user credentials before you start, they are randomly generated. When this happens, the `helm update` command does not rotate the credentials, to prevent breaking the database.
+If you do not configure the user credentials before you start, they are randomly
```

Comment:
I discussed this issue with Pawel, apparently there is an issue here with the most recent timescale helm charts not being published on `charts.timescale.com`. I have opened an issue there https://github.com/timescale/timescaledb-kubernetes/issues/324.

Reply:
@Loquacity I have marked this as unresolved - I don't think that this has been addressed. These instructions do not work.

Reply:
@JamesGuthrie I don't fully understand the problem? Why do we have a hard requirement on the last version of the timescaledb charts to be published? Is it because we released changes on our side that rely on the latest versions of timescaledb charts that are not published?

Reply:
@ramonguiu Honestly I don't really know. As far as I can tell this is purely about how the timescaledb helm charts work. When following these instructions with the chart which is currently published on `charts.timescale.com` it doesn't work. It's possible that by correctly configuring the `values.yaml` (as alluded to in the instructions), it would be possible to make it work. I spent about 15 minutes trying to make that work and decided to give up. When I discussed it with Pawel, he said that there were some changes which make these steps work "out of the box" in the latest helm charts. When I tried these steps with the helm chart v0.10.0 (which I got directly from the github repo) it worked perfectly.